mappings/UofM.py

```diff
@@ -75,6 +70,40 @@ def formatIdentifiers(self):
                     return newISBNList
                 
         return self.record.identifiers
+    
+    def formatSubjects(self):
+        subjectList = []
+
+        if '|' in self.record.subjects:
+            subjectList = self.record.subjects.split('|')
+            newSubjectList = [f'{subject}||' for subject in subjectList] 
+            return newSubjectList
+        else:
+            subjectList.append(f'{self.record.subjects}||')
+            return subjectList
+    
+    def formatRights(self):
+        # Parse rights codes
+        rightsElements = self.record.rights.split('|') if self.record.rights else [''] * 5
+        rightsMetadata = rightsElements[0]
+        licenseMeta = ''
+        statementMeta = ''
+        print(rightsMetadata)
+
+        if rightsMetadata == 'in copyright':
+            licenseMeta = 'in_copyright'
+            statementMeta = 'In Copyright'
+        if rightsMetadata == 'public domain':
+            licenseMeta = 'public_domain'
+            statementMeta = 'Public Domain'
```

Comment:
are there any other possible values for rightsMetadata?

Reply:
There are rights_statements and rights_date which are not necessary to have and are not currently available on Airtable.